Formula/op25.rb

```diff
@@ -0,0 +1,64 @@
+class Op25 < Formula
+  desc "Software-defined analyzer for APCO P25 signals"
+  homepage "https://osmocom.org/projects/op25/wiki"
+  url "git://op25.osmocom.org/op25.git"
+
+  depends_on "cmake" => :build
+  depends_on "cppunit" => :build
+  depends_on "swig" => :build
+  depends_on "gnuradio"
+  depends_on "gr-osmosdr"
+  depends_on "itpp"
+
+  def install
+    # TODO: apply patch if gnuradio >= 3.8?
```

Comment:
Currently expects to build against `gnuradio@3.7` as explained in https://git.osmocom.org/op25/commit/?id=c26b51e0c1b5fd9eee0c5b13dc137050cd0c988b, refs https://github.com/Homebrew/homebrew-core/pull/49733